android/PartySampleNetworkCommon/lib/NetworkManager.cpp

```diff
@@ -169,10 +169,10 @@ NetworkManager::CreateAndConnectToNetwork(
     PartyNetworkConfiguration cfg = {};
 
     // Setup the network to allow the maximum number of single-device players of any device type
-    cfg.maxDeviceCount = c_maxNetworkConfigurationMaxDeviceCount;
+    cfg.maxDeviceCount = 29;
```

Comment:
nit: maybe one way this could be improved would be by defining that `29` as a sample constant somewhere at the top like 
```
// For the sample, pick an arbitrary number of devices to allow in the network. Only needs to be less than Party's max device limits
constexpr uint8_t c_maxSampleNetworkDeviceCount = 29;
static_assert(c_maxSampleNetworkDeviceCount <= c_maxNetworkConfigurationMaxDeviceCount);
```

and then here you can assign that value to these cfgs
```
    cfg.maxDeviceCount = c_maxSampleNetworkDeviceCount; // 1 unique user per device
    cfg.maxDevicesPerUserCount = 1;
    cfg.maxEndpointsPerDeviceCount = 1;
    cfg.maxUserCount = c_maxSampleNetworkDeviceCount; // 1 unique user per device
```